From 90a5986816e16000948402ac6e850f693c8510ed Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Fri, 14 Aug 2020 13:12:29 +0700
Subject: [PATCH] core: arm: plat-rzg: Disable initial console

When we use console(scif) driver, we will get conflict with scif driver
of untrusted OS if when using Suspend To RAM feature.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 core/arch/arm/plat-rzg/main.c | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/core/arch/arm/plat-rzg/main.c b/core/arch/arm/plat-rzg/main.c
index 921da8a1..e8580336 100644
--- a/core/arch/arm/plat-rzg/main.c
+++ b/core/arch/arm/plat-rzg/main.c
@@ -127,6 +127,11 @@ const struct thread_handlers *generic_boot_get_handlers(void)
 
 void console_init(void)
 {
-    scif_uart_init(&console_data, CONSOLE_UART_BASE);
-    register_serial_console(&console_data.chip);
+      /* We won't init console(scif) for OPTEE-os, this driver will conflict with 
+       * scif driver of un-trusted OS when using Suspend To RAM feature. 
+       */
+      /*
+       * scif_uart_init(&console_data, CONSOLE_UART_BASE);
+       * register_serial_console(&console_data.chip);
+       */ 
 }
-- 
2.17.1

